### PR TITLE
handle rotationinterval updates in reconciler

### PR DESCRIPTION
**Pull request by rahulsawra98**
# Description

Fixes #

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR requires documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code.
* [ ] I have commented my code, particularly in hard-to-understand areas.
* [ ] I have tested it for all user roles.
* [ ] I have added all the required unit test cases.

## Does this PR introduce a breaking change for other components like worker-operator?

```release-note

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Image URL to use all building/pushing image targets
-IMG ?= aveshasystems/kubeslice-controller:latest
+IMG ?= aveshasystems/kubeslice-controller:latest-v2
```
Comment (rajendra-avesha) [inline Makefile:2]: Please add descriptions for PRs

File: service/slice_config_webhook_validation.go [hunk 2]
```diff
@@ -159,6 +163,35 @@ func validateRenewNowInSliceConfig(ctx context.Context, sliceConfig *controllerv
...
+		return nil, nil
+	}
+	// change detected
+	vpnKeyRotation := controllerv1alpha1.VpnKeyRotation{}
+	exists, _ := util.GetResourceIfExist(ctx, types.NamespacedName{
+		Namespace: sliceConfig.Namespace,
+		Name:      sliceConfig.Name,
+	}, &vpnKeyRotation)
+	if exists {
+		vpnKeyRotation.Spec.RotationInterval = sliceConfig.Spec.RotationInterval
```
Comment (rajendra-avesha) [inline service/slice_config_webhook_validation.go:179]: Do we need to  check for the valid of min and max or will be taken care please add the tests for the same

File: Makefile (hunk 1 above)
Comment (narmidm) [reply Makefile:2]: remove this

**Review by narmidm — APPROVED** [review]

File: service/slice_config_webhook_validation.go [hunk 3]
```diff
@@ -24,6 +24,7 @@ import (
 	"strings"
 	"time"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
[the remark this answers is not in the document]
Comment (rahulsawra98) [reply service/slice_config_webhook_validation.go:27]: Done!

File: service/slice_config_webhook_validation.go (hunk 2 above)
Comment (rahulsawra98) [reply service/slice_config_webhook_validation.go:179]: should already be handled!

**Pull request merged**